packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -298,6 +300,36 @@ class _FocusState extends State<Focus> {
     _initNode();
   }
 
+  bool _handleOnKey(FocusNode node, RawKeyEvent event) {
```

Comment:
i don't have any solutions here but it would be cleaner if this class didn't know about shortcuts at all.

Reply:
OK, I successfully moved this into the `Shortcuts` widget from the `Focus` widget.